src/miner.cpp

```diff
@@ -640,10 +640,11 @@ void static BitcoinMiner()
                     LogPrintf("ZcashMiner:\n");
                     LogPrintf("proof-of-work found  \n  hash: %s  \ntarget: %s\n", pblock->GetHash().GetHex(), hashTarget.GetHex());
 #ifdef ENABLE_WALLET
-                    if (ProcessBlockFound(pblock, *pwallet, reservekey)) {
```

Comment:
Hope no one minds that I moved the open brace to common code, so that the braces match up in my editor.